libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -182,9 +182,10 @@ AP_GPS_ERB::_parse_gps(void)
             state.time_week_ms    = _buffer.stat.time;
             state.time_week       = _buffer.stat.week;
         }
+        Debug("Message ERB STAT num_sats= %u", _buffer.stat.satellites);
```

Comment:
The changes in this file seam to be unintended

Reply:
I had put that in there when I was testing the Reach as well when they weren't sending the hdop and satellites in non-RTK mode. I can remove it.